bitnami/spark/values-production.yaml

```diff
@@ -0,0 +1,237 @@
+## Global Docker image parameters
+## Please, note that this will override the image parameters, including dependencies, configured to use the global value
+## Current available global Docker image parameters: imageRegistry and imagePullSecrets
+##
+# global:
+#   imageRegistry: myRegistryName
+#   imagePullSecrets:
+#     - myRegistryKeySecretName
+
+## Bitnami Spark image version
+## ref: https://hub.docker.com/r/bitnami/spark/tags/
+##
+image:
+  registry: docker.io
+  repository: 'bitnami/spark'
+  tag: 2.4.3
+  ## Specify a imagePullPolicy
+  ## Defaults to 'Always' if image tag is 'latest', else set to 'IfNotPresent'
+  ## ref: http://kubernetes.io/docs/user-guide/images/#pre-pulling-images
+  ##
+  pullPolicy: IfNotPresent
+
+  ## Pull secret for this image
+  # pullSecrets:
+  #   - myRegistryKeySecretName
+
+
+## Spark Components configuration
+##
+master:
+  ## Spark master specific configuration
+  webPort: 8080
+  clusterPort: 7077
+
+  ## Set the master daemon memory limit.
+  # daemonMemoryLimit:
+
+  ## Use a string to set the config options for in the form "-Dx=y"
+  # configOptions:
+
+  ## Set to true if you would like to see extra information on logs
+  ## It turns BASH and NAMI debugging in minideb
+  ## ref:  https://github.com/bitnami/minideb-extras/#turn-on-bash-debugging
+  debug: false
+  ## StatefulSet controller supports automated updates. There are two valid update strategies: RollingUpdate and OnDelete
```

Comment:
We are not using a StatefulSet, but a Deployment

Reply:
Changed.